Makefile

```diff
@@ -65,7 +69,7 @@ fmt-go:
 fmt: $(fmt-targets)
 
 # Lint code, commits and documentation.
-lint-targets := lint-go lint-git lint-md lint-changelog
+lint-targets := lint-go lint-git lint-md lint-changelog check-docs
```

Comment:
Can you change the ci-lint workflow to actually run this shiny new `check-docs` target (maybe rename to `lint-docs` to be consistent)? See `.github/workflows/ci-lint.yml`.